bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/handler/OpenSprinklerDeviceHandler.java

```diff
@@ -57,4 +72,29 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         // nothing to do here
     }
 
+    @Override
+    public void updateChannels() {
+        ChannelUID currentDraw = new ChannelUID(thing.getUID(), "currentDraw");
+        ThingBuilder thingBuilder = editThing();
+        Channel currentDrawChannel = ChannelBuilder.create(currentDraw, "Number:ElectricCurrent")
+                .withType(new ChannelTypeUID(BINDING_ID, SENSOR_CURRENT_DRAW)).withLabel("Current Draw")
+                .withDescription("Provides the current draw.").build();
```

Comment:
Please move this logic to line 86, just before you would need to use it. No reason to create this variable without knowing if you need it or not.

Reply:
Done.